### PR TITLE
Update NVidia.xml

### PR DESCRIPTION
**Pull request by koops76**

### CODE REVIEW

File: src/chrome/content/rules/NVidia.xml [hunk 1]
```diff
@@ -136,72 +79,19 @@
...
+	<target host="omniture.nvidia.com" />
 	<target host="partners.nvbugs.nvidia.com" />
 	<target host="nvdeveloper.nvidia.com" />
-	<!--target host="nvidianews.nvidia.com" /-->
 	<target host="partners.nvidia.com" />
 	<target host="es.pforce.nvidia.com" />
 	<target host="redeem.nvidia.com" />
-	<!--target host="registration.nvidia.com" /-->
 	<target host="shield.nvidia.com" />
-	<target host="store.nvidia.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/NVidia.xml]: Why did you remove this host?

File: src/chrome/content/rules/NVidia.xml [hunk 2]
```diff
@@ -60,64 +45,22 @@
...
-			- www from \w+.fls.doubleclick.net
-			- www from media.fastclick.net *
-
-	* Secured by us
 -->
 
 <ruleset name="nVidia.com (partial)">
-	<!--	Direct rewrites:
-				-->
+	<target host="nvidia.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/NVidia.xml]: Please update the `Nonfunctional subdomains` and `Problematic subdomains` lists

File: src/chrome/content/rules/NVidia.xml (hunk 1 above)
Comment (koops76) [reply src/chrome/content/rules/NVidia.xml]: Probably because it's broken in some way.

Comment (koops76) [reply src/chrome/content/rules/NVidia.xml]: I can't see any breakage now, reenabling.

**Comment by J0WI** [comment]
Travis exposed an error. Please also fix https://github.com/EFForg/https-everywhere/pull/11548#discussion_r137360630

**Comment by koops76** [comment]
ping @J0WI.

**Comment by J0WI** [comment]
This is still blocked by https://github.com/EFForg/https-everywhere/pull/11548#discussion_r137360630
